data-prepper-logstash-configuration/src/main/java/org/opensearch/dataprepper/logstash/mapping/AbstractLogstashPluginAttributesMapper.java

```diff
@@ -66,6 +63,12 @@ else if(defaultSettings.containsKey(logstashAttributeName)) {
                     }
                 });
 
+        for(Map.Entry<String, Object> defaultSetting: defaultSettings.entrySet()) {
```

Comment:
Should this code block happen first or do we want the defaults to override everything else?

Reply:
The condition on line 67 prevents overriding existing values.

Reply:
If we applied the defaults first, could we eliminate the condition check?

Reply:
That also would work, but I'm ok with either approach.

Reply:
The defaults won't override if an `attributeValue` already exists.